lib/maze/option/parser.rb

```diff
@@ -138,10 +138,10 @@ def parse(args)
             text 'Logging options:'
 
             opt Option::LOG_REQUESTS,
-                "Log a list of received requests in the event of test failure",
+                "Log a list of received requests in the event of test failure.  Defaults to true if the BUILDKITE environment variable is set",
                 short: :none,
                 type: :boolean,
-                default: true
+                default: false
```

Comment:
Is there a downside to keeping the default `true`?

Reply:
It upsets fewer people setting it false.  Most people have asked for the logging of requests to disappear from the console by default (and not just in today's internal poll).